cmd/otelcontribcol/builder-config.yaml

```diff
@@ -465,6 +466,7 @@ replaces:
   - github.com/open-telemetry/opentelemetry-collector-contrib/connector/roundrobinconnector => ../../connector/roundrobinconnector
   - github.com/open-telemetry/opentelemetry-collector-contrib/connector/routingconnector => ../../connector/routingconnector
   - github.com/open-telemetry/opentelemetry-collector-contrib/connector/servicegraphconnector => ../../connector/servicegraphconnector
+  - github.com/open-telemetry/opentelemetry-collector-contrib/connector/signaltometricsconnector => ../../connector/signaltometricsconnector
```

Comment:
Same, let's keep it our of the distro for now. It should be added here in the last PR.

ref: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#adding-new-components